src/js/taskFunction.js

```diff
@@ -76,8 +76,7 @@ class Todolist {
   };
 
   addtask = (description, completed, index) => {
-    index = 1;
-    index = this.taskData.length;
+    index = 1 + this.taskData.length;
     const newtask = new Tasks(description, completed, index);
     this.taskData.push(newtask);
     if (typeof window !== 'undefined') {
```

Comment:
- Dear @otmanTR, Good job making most of the changes, Yet as the previous code reviewer mentioned we need to preserve the idea of indexing. Every task should be set to the latest length of the array after the task is inserted. Right now I can see it works the first time, But it failed when I add more tasks. Also, this behavior needs to be preserved on a task delete. (if you have 3 tasks and we delete the middle one the remaining tasks should have index=1, index=2 respectively). 

https://user-images.githubusercontent.com/69077061/215025206-19b07b21-31cf-4bf2-b3df-70d4fd8bae46.mp4